Cargo.toml

```diff
@@ -21,32 +21,55 @@ features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = []
-docs = ["unstable", "attributes"]
-unstable = ["broadcaster"]
-attributes = ["async-attributes"]
+default = [
+  "std",
+  "async-task",
+  "crossbeam-deque",
+  "futures-timer",
+  "kv-log-macro",
+  "log",
+  "mio",
+  "mio-uds",
+  "num_cpus",
+  "pin-project-lite",
+]
+docs = ["unstable"]
+unstable = ["default", "broadcaster"]
+std = [
+  "async-macros",
+  "crossbeam-channel",
+  "crossbeam-utils",
+  "futures-core",
+  "futures-io",
+  "futures-timer",
```

Comment:
Timers should also belong in `runtime` since they need a background thread... which is a runtime :)